### PR TITLE
Add Derry/Londonderry file

### PR DESCRIPTION
**Pull request by diarmaidm**
This is the beginning of the Derry/Londonderry file :umbrella: 

Please provide feedback on:

- What do you want to know about?
- We have a nodeschool chapter http://nodeschool.io/foyle-valley/

### CODE REVIEW

File: diarmaidm-Derry.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Restaurants
+
+### Local Attractions
+
+- The Walls
+- The Quiet Man Whiskey
+
+### Remote Attractions
+
```
Comment (readiu) [inline diarmaidm-Derry.md]: Where is this town?

**Comment by jetgo1000** [comment]
I want to go to nodeschool!

File: diarmaidm-Derry.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Derry, Northern Ireland
+
+### Restaurants
+
+### Local Attractions
+
+- The Walls
+- The Quiet Man Whiskey
```
Comment (nysertxs) [inline diarmaidm-Derry.md]: id like to try it 🍻 🍺  😄

File: diarmaidm-Derry.md (hunk 1 above)
Comment (diarmaidm) [inline diarmaidm-Derry.md]: Its in Northern Ireland

File: diarmaidm-Derry.md (hunk 2 above)
Comment (diarmaidm) [inline diarmaidm-Derry.md]: It is really nice, recommend it

File: diarmaidm-Derry.md (hunk 1 above)
Comment (diarmaidm) [inline diarmaidm-Derry.md]: Northern Ireland https://www.google.com/maps/d/viewer?mid=1b59aw1eXLd6wxCcnrtmy-UWRAak

Comment (diarmaidm) [inline diarmaidm-Derry.md]: I added a link to google maps in b5fa972

File: diarmaidm-Derry.md (hunk 2 above)
Comment (diarmaidm) [inline diarmaidm-Derry.md]: I put some more info in 4e34604

**Pull request merged**